fileWithPRComments.js

```diff
@@ -0,0 +1,8 @@
+/* global QamineUtils */
+
+"use strict";
+
+(function($, window) {
```

Comment:
Codacy has a fix for the issue: [Insert `·`](https://app.codacy.com/gh/prodcodacytestuser/CheckPrCommentsPublicProjectProd/pullRequest?prid=9341204)
```suggestion
(function ($, window) {
```